public/locales/en/modules/dlspeed.json

```diff
@@ -1,7 +1,7 @@
 {
   "descriptor": {
     "name": "Download Speed",
-    "description": "Show the current download speed of supported services"
+    "description": "Displays the Download and Upload speed on supported devices."
```

Comment:
Should be intégrations / apps with untrgrwtiins

Reply:
I based this of how it's named in the docs, docs will have to be changed.